vaadin-avatar-flow-parent/vaadin-avatar-flow/src/test/java/com/vaadin/flow/component/avatar/tests/AvatarGroupTest.java

```diff
@@ -21,56 +21,52 @@
 import com.vaadin.flow.component.avatar.AvatarGroupVariant;
 import com.vaadin.tests.ThemeVariantTestHelper;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 
-/**
- * @author Vaadin Ltd.
- */
 public class AvatarGroupTest {
 
-    private AvatarGroup avatarGroup = new AvatarGroup();
-    private AvatarGroupItem avatarGroupItem = new AvatarGroupItem();
-    private AvatarGroupItem avatarGroupItem2 = new AvatarGroupItem("Foo Bar");
-
-    private String name = "foo bar";
-    private String abbr = "fb";
-    private String imgUrl = "https://vaadin.com/";
-    private Integer colorIndex = 3;
+    private AvatarGroup avatarGroup;
+    private AvatarGroupItem avatarGroupItem1;
+    private AvatarGroupItem avatarGroupItem2;
 
-    private List<AvatarGroupItem> items = new ArrayList<>();
+    @Before
+    public void setUp() {
```

Comment:
**nit:** `init` seems to be a more common naming.

```suggestion
    public void init() {
```

Reply:
As discussed, `setup` is more common for unit tests in general, while `init` is more common for ITs, probably because `setup` is already used in the class hierarchy.